src/pages/iou/IOUModal.js

```diff
@@ -346,8 +346,7 @@ class IOUModal extends Component {
             Math.round(this.state.amount * 100),
             this.props.iou.selectedCurrencyCode,
             this.props.currentUserPersonalDetails.login,
-            OptionsListUtils.addSMSDomainIfPhoneNumber(this.state.participants[0].login),
-            lodashGet(this.state.participants[0], 'firstName', ''),
+            selectedParticipants[0],
```

Comment:
` selectedParticipants[0]` or `this.state.participants[0]`

Reply:
selectedParticipants[0] works :)